master/internal/api_model.go

```diff
@@ -157,8 +157,11 @@ func (a *apiServer) PatchModel(
 	currLabels := strings.Join(currModel.Labels, ",")
 	if req.Model.Labels != nil {
 		var reqLabelList []string
-		for i := 0; i < len(req.Model.Labels); i++ {
-			reqLabelList = append(reqLabelList, req.Model.Labels[i].Value)
+		for _, el := range req.Model.Labels.Values {
+			sval := el.String()
+			if sval[:13] == "string_value:" {
+				reqLabelList = append(reqLabelList, sval[14:len(sval)-1])
+			}
```

Comment:
would something like this work?

```suggestion
            if _, ok := el.GetKind().(*structpb.Value_StringValue); ok {
                reqLabelList = append(reqLabelList, el.GetStringValue());
            }
```

Reply:
I like this change, just trying to catch up now 😅